arch/arm/src/imxrt/imxrt_flexpwm.c

```diff
@@ -873,21 +905,38 @@ static int pwm_set_output(struct pwm_lowerhalf_s *dev, uint8_t channel,
   regval |= MCTRL_CLDOK(1 << shift);
   putreg16(regval, priv->base + IMXRT_FLEXPWM_MCTRL_OFFSET);
 
-  /* Write width to value register 3 and enable output A */
+  if ((channel - 1) % 2)
```

Comment:
Optional
```suggestion
  if ((channel - 1) % 2 != 0)
```

Reply:
Do I need to fix something in order to merge PR?

Reply:
No. Usually the reply "I would prefer to keep this as is" would be a sign that the author processed the comment.